### PR TITLE
Fix and extend input functionality

### PR DESCRIPTION
**Pull request by ammarcsj**
Bug fix: Resolved issue with additional columns

- Modified default value for additional columns
- Implemented proper handling of string input from users
- This addresses the bug reported in issue #47

Extended input functionality:

- Added Alphadia configuration

### CODE REVIEW

File: directlfq/utils.py [hunk 1]
```diff
@@ -256,8 +256,10 @@ def add_columns_to_lfq_results_table(lfq_results_df, input_file, columns_to_add)
     all_columns = columns_to_add + [protein_column_input_table] + standard_columns_for_input_type
     all_columns = filter_columns_to_existing_columns(all_columns, input_file)
 
-    input_df = pd.read_csv(input_file, sep="\t", usecols=all_columns).drop_duplicates(subset=protein_column_input_table)
     lfq_results_df = lfq_results_df[[x is not None for x in lfq_results_df[config.PROTEIN_ID]]]
+    if len(all_columns) == 1: #if there are no columns to add, return the original dataframe
```
Comment (mschwoer) [inline directlfq/utils.py:260]: I see some discrepancy between the comment and the logic above:
`columns_to_add + [protein_column_input_table] + standard_columns_for_input_type`

File: directlfq/dashboard_parts.py [hunk 2]
```diff
@@ -327,6 +327,8 @@ def run_pipeline(self, *args):
         file_of_proteins_for_normalization = None if self.protein_subset_for_normalization_file.value == '' else self.protein_subset_for_normalization_file.value
         num_cores = None if self.num_cores_vals.value == -1 else self.num_cores_vals.value
         yaml_filt_dict_path = None if self.yaml_filt_dict_path.value == '' else self.yaml_filt_dict_path.value
+        if type(additional_headers) == str: #the user will enter a string with semicolon separated values
```
Comment (mschwoer) [inline directlfq/dashboard_parts.py:330]: can this be any other type?

File: directlfq/utils.py (hunk 1 above)
Comment (ammarcsj) [reply directlfq/utils.py]: so both the columns to add (columns specified by the user) and the standard columns (columns added by default) can be empty lists. I now made the if statement more explicit.

`    if (len(columns_to_add) == 0) and (len(standard_columns_for_input_type)==0) : #if there are no columns to add and no additional standard columns defined return the original dataframe
        return lfq_results_df `

File: directlfq/dashboard_parts.py (hunk 2 above)
Comment (ammarcsj) [reply directlfq/dashboard_parts.py:330]: any user input will be converted to a string. If there is no user input, there will be None

Comment (mschwoer) [reply directlfq/dashboard_parts.py:330]: so this could be `if additional_headers is not None:`?

File: directlfq/utils.py (hunk 1 above)
Comment (mschwoer) [reply directlfq/utils.py]: sounds good, I think you could even drop the comment then as the code reads almost as good as plain text ;-)

Comment (ammarcsj) [reply directlfq/utils.py]: yess ☺️

**Comment by ammarcsj** [comment]
thx!

**Pull request merged**